src/test/java/seedu/address/logic/commands/sort/SortKeyTest.java

```diff
@@ -0,0 +1,75 @@
+package seedu.address.logic.commands.sort;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import org.junit.jupiter.api.Test;
+
+public class SortKeyTest {
+
+    @Test
+    public void isValidSortKey() {
+
+        // null sort label
+        assertThrows(NullPointerException.class, () -> SortKey.isValidSortKey(null));
+
+        assertFalse(SortKey.isValidSortKey("")); // empty String
+        assertFalse(SortKey.isValidSortKey(" ")); // spaces only
+        assertFalse(SortKey.isValidSortKey(" deadline")); // first character whitespace
+        assertFalse(SortKey.isValidSortKey("123")); // digits
+        assertFalse(SortKey.isValidSortKey("asc")); // valid sort order
+        assertFalse(SortKey.isValidSortKey("desc")); // valid sort order
+        assertFalse(SortKey.isValidSortKey("dl")); // abbreviation of deadline
+        assertFalse(SortKey.isValidSortKey("p")); // abbreviation of priority
+        assertFalse(SortKey.isValidSortKey("n")); // abbreviation of name
```

Comment:
Might be good to abstract this out into its own file, like what was discussed in the chat – so that we can avoid magic strings!

I think you can also use `Asset::assertAllFalse` for this!